shared/git/new-repo/index.tsx

```diff
@@ -105,63 +105,65 @@ class NewRepo extends React.Component<Props, State> {
     return (
       <Kb.ScrollView>
         <Kb.Box style={styles.container}>
-          {!!this.props.error && (
-            <Kb.Box style={styles.error}>
-              <Kb.Text type="Body" negative={true}>
-                {this.props.error.message}
-              </Kb.Text>
-            </Kb.Box>
-          )}
-          <Kb.Text type="Header" style={{marginBottom: 27}}>
-            New {this.props.isTeam ? 'team' : 'personal'} git repository
-          </Kb.Text>
-          <Kb.Icon
-            type={this.props.isTeam ? 'icon-repo-team-add-48' : 'icon-repo-personal-add-48'}
-            style={styles.addIcon}
-          />
-          <Kb.Text type="Body" style={{marginBottom: 27}}>
-            {this.props.isTeam
-              ? 'Your repository will be end-to-end encrypted and accessible by all members in the team.'
-              : 'Your repository will be encrypted and only accessible by you.'}
-          </Kb.Text>
-          {this.props.isTeam && (
-            <Kb.Dropdown
-              items={this._makeDropdownItems()}
-              selected={this._makeDropdownItem(this.state.selectedTeam)}
-              onChanged={this._dropdownChanged}
-              style={styles.dropdown}
+          <Kb.Squeeze enable={Styles.isTablet}>
```

Comment:
Are we planning to make modals a thing on iPad? if so, seems like this would rather be that than a squeeze.